### ISSUE TITLE
ReferenceError: Can't find variable: Highcharts

### ISSUE THREAD

**Issue by zejacobi** [nfantone/highcharts-export-server#49]
I got this error on Win10, while trying to create a chart render via the request module in node. I was using this package installed globally, as per the NPM  instructions.

Full Error Trace:

```
Highcharts.options.parsed
Highcharts.cb.parsed
Highcharts.customCode.parsed
ERROR: ReferenceError: Can't find variable: Highcharts
TRACE:
 -> undefined: 61
 -> : 140

  phantomjs://code/highcharts-convert.js:160 in onError
```

Seems related to an issue on [Stack Overflow](http://stackoverflow.com/questions/38294099/highcharts-and-phantomjs-error-cant-find-variable-highcharts) and [issue #18 in the original repo](https://github.com/highcharts/highcharts-export-server/issues/18)

**Comment by nfantone** [comment]
Thank you very much for this. Didn't realize that issue existed in the `upstream`. Already merged and will release soon.

**Comment by nfantone** [comment]
Done. Published `2.2.1` [to npm](https://www.npmjs.com/package/highcharts-phantomjs).

**Comment by suhaboncukcu** [comment]
I got the same error with 2.2.1 (I assume)

```
  phantomjs://code/highcharts-convert.js:160 in onError
/srv/users/serverpilot/phantomjs/node_modules/highcharts-phantomjs/lib/resources.json
Highcharts.options.parsed
Highcharts.customCode.parsed
ERROR: ReferenceError: Can't find variable: Highcharts
TRACE:
 -> undefined: 61
 -> : 140
```
